### PR TITLE
Add additional attributes to Hit per issue #16

### PR DESCRIPTION
**Pull request by seeingidog**
Needed part of these attributes, went ahead and added the rest to satisfy issue. Let me know if you need any changes before merging. Thanks!

### CODE REVIEW

**Comment by mdp** [comment]
Thanks Ian, I'll add this into the next version, which I'm hoping to be able to get out tomorrow.

File: spec/fake_responses/get_hit.xml [hunk 1]
```diff
@@ -12,10 +12,10 @@
     <CreationTime>2009-06-25T04:21:17Z</CreationTime>
     <Title>Write a twitter update</Title>
     <Description>Simply write a twitter update for me</Description>
-    <Question>&lt;ExternalQuestion xmlns="http://mechanicalturk.amazonaws.com/AWSMechanicalTurkDataSchemas/2006-07-14/ExternalQuestion.xsd"&gt;
-	&lt;ExternalURL&gt;http://s3.amazonaws.com/mpercival.com/newtweet.html?id=foo&lt;/ExternalURL&gt;	
-	&lt;FrameHeight&gt;400&lt;/FrameHeight&gt;
-&lt;/ExternalQuestion&gt;
+    <Question><ExternalQuestion xmlns="http://mechanicalturk.amazonaws.com/AWSMechanicalTurkDataSchemas/2006-07-14/ExternalQuestion.xsd">
```
Comment (mdp) [inline spec/fake_responses/get_hit.xml]: I"m not actually sure that this is the case. I remember MTurk was returning the content of Question as escape XML(I have no idea why). Can you confirm that this is still the case?

Comment (seeingidog) [inline spec/fake_responses/get_hit.xml]: You're correct, I made a silly assumption here. I've reverted the fake response locally and tests are passing after unescaping the XML. I'll push that up soon, Thanks!

**Comment by seeingidog** [comment]
Should be good now.

**Pull request merged**